### ISSUE TITLE
Library Explorer accessibility: No keyboard focus on floating controls

### ISSUE THREAD

**Issue by lephemere** [internetarchive/openlibrary#5037]
In the Library Explorer, it is not possible to focus the `floating-controls` section with the keyboard.

Parent issue #4908.

### Evidence / Screenshot (if possible)
![image](https://user-images.githubusercontent.com/73437497/114388600-31ecce00-9b94-11eb-822e-8739dd10b3c0.png)

### Relevant url?

https://openlibrary.org/explore

### Details

- **Logged in (Y/N)?** N
- **Browser type/version?** FF87
- **Operating system?** 
- **Environment (prod/dev/local)?** prod/dev

### Stakeholders

@bpmcneilly

**Comment by cdrini** [comment]
`<div class="chunky-icon">` should be change to a `<button>` :+1:
